api/libproject/devstart.rb

```diff
@@ -2016,7 +2016,7 @@ def invalidate_rdr_export(cmd_name, *args)
   end
 
   get_user_confirmation(
-    "RDR export invalidation should rarely be used; for backfilling workspace data you " +
```

Comment:
@calbach could you confirm that this change is correct?

Reply:
Thanks, yes - left in accidentally.